packages/opentelemetry-shim-opentracing/src/shim.ts

```diff
@@ -331,13 +332,28 @@ export class SpanShim extends opentracing.Span {
     return this;
   }
 
-  /*
-   * Returns the underlying {@link types.Span} that the shim
+  /**
+   * Returns the underlying {@link api.Span} that the shim
    * is wrapping.
    */
   getSpan(): api.Span {
     return this._span;
   }
+
+  private static _mapErrorTag(
+    value: SpanAttributeValue
+  ): SpanStatusCode | undefined {
+    switch (value) {
+      case true:
+      case 'true':
+        return SpanStatusCode.ERROR;
+      case false:
+      case 'false':
+        return SpanStatusCode.OK;
+      default:
+        return;
```

Comment:
I would prefer to return `SpanStatusCode.UNSET` here to avoid problem in the future

Reply:
Being explicit makes sense. Currently, any error tag that's not "true" or "false" will get set as an attribute. (For example, in the test suite, error: whoopsie will result in UNSET status code and a span attribute `error: whoopsie`) Do you think that still makes sense? I ask because the downstream logic short-circuits setting the attribute if a defined status code is returned from here, but I can easily change that.

Reply:
Updated in new commits